### ISSUE TITLE
The noto-fonts package is very outdated

### ISSUE THREAD

**Issue by Comevius** [clearlinux/distribution#909]
The [noto-fonts](https://github.com/clearlinux-pkgs/noto-fonts) package is outdated (**2015.09.29.license.adobe**) and is missing a lot of new fonts, such as Noto Sans Mono.

The newer releases switched from Apache License 2.0 to the SIL Open Font License, which is the same license that [noto-cjk](https://github.com/clearlinux-pkgs/noto-cjk/blob/master/noto-cjk.spec) (2017.06.01.serif.cjk.1.1) and [noto-emoji](https://github.com/clearlinux-pkgs/noto-emoji/blob/master/noto-emoji.spec) (2017.03.10.color.emoji.ary.1) uses, so that should not be a problem.

Ideally you should at least get us [**v2017-09-19-phase3-second**](https://github.com/googlefonts/noto-fonts/releases/tag/v2017-09-19-phase3-second), which is tagged for release, but you can also pick a newer commit from master branch, like many other Linux distributions do.

**Comment by mrkz** [comment]
Hello @Comevius, thanks for your report.

I have pushed an update which should ship current master branch on the noto-fonts repository ( commit [87a88abd](https://github.com/googlefonts/noto-fonts/commit/87a88abdee6cf193c8bb7c883376a23ba438251b)).

This should be available soon-ish (hopefully on release **30730** or later).

Additionally, I have updated the noto-emoji fonts as well which include some emojis not present on previous release we were shipping.

:tada:
